template/accordion/accordion-group.html

```diff
@@ -1,4 +1,4 @@
-<div class="panel panel-default">
+<div class="panel panel-{{panelClass || 'default'}}">
```

Comment:
There are some users of the library who use it with custom CSS or other CSS libraries - I think we should try to be less opinionated here for minimal pain.